supporting_content/attributes/column_naming_convention.md

```diff
@@ -16,3 +16,57 @@ Examples of column names found in provider datasets:
 - Group wanted to get naming convention adopted broadly for provider columns as well, to simplify reporting and analysis by practitioners
 - Different conventions like camel case, snake case, and title case were considered
 - Discussion included different columns naming conventions used by databases
+
+### How to handle custom columns
+
+- Providers need to add additional details that are not available in FOCUS columns to support additional FinOps scenarios.
+- FOCUS needs to allow the addition of these columns without the risk of collision with future FOCUS columns.
+- Considerations:
+  - **Placement (prefix vs. suffix):**
+    - A prefix would be easier to identify as a custom column even with limited space.
+    - A suffix would require expanding the column or explicitly looking at the end of the column name, which can be difficult is some client tools where the end of the name gets cut off.
+    - BEST: **Prefix**.
+  - **Separator character:**
+    - Should be easily identifiable as a separator.
+      - Common: `_`, `-`, `.`, `:`. `@`, `/`, `~`, `|`, `+`, `=`, `;`
+      - Uncommon: `$`, `&`, `#`, `!`, `?`, `*`
+    - Should avoid characters that don't work in most tools or programming languages, since they would need to divert from the spec.
+      - Can be used in variable names: `_`
+      - Can be logically handled via nesting: `.`, `/`, `:`
+      - Limited support: `-`, `$`, `@`, `#`. `~`
+    - Prefer characters that are commonly used in spreadsheet and database column names.
+      - Common: `_`, `.`, `/`
```

Comment:
Forward slashes and periods are generally frowned up on in database columns, as you have to put brackets around them to make them work in things like SQL queries.... Underscores are best.  Should the spec be more specific and/or stronger in its recommendation to avoid these?

Reply:
We're not recommending them at all. We explicitly say you can't. This is just supporting content covering the thoughts that led up to the spec. I think we should be good from that perspective.